images-list

```diff
@@ -616,6 +624,8 @@ longhornio/longhorn-ui rancher/mirrored-longhornio-longhorn-ui v1.2.6
 longhornio/longhorn-ui rancher/mirrored-longhornio-longhorn-ui v1.3.0
 longhornio/longhorn-ui rancher/mirrored-longhornio-longhorn-ui v1.3.1
 longhornio/longhorn-ui rancher/mirrored-longhornio-longhorn-ui v1.3.2
+longhornio/longhorn-ui rancher/mirrored-longhornio-longhorn-ui v1.4.0
+longhornio/support-bundle-kit rancher/mirrored-longhornio-support-bundle-kit v0.0.16
```

Comment:
Support bundle kit is updated to newer version https://github.com/longhorn/longhorn/blob/f48a8ee582691da148a02157ca653a093e92552b/chart/values.yaml#L46

Reply:
Thanks for reminding. I've updated the `support-bundle-kit` version as `v0.0.17`.